_config.yml

```
@@ -15,6 +15,8 @@ download_file_names:
   deb-headless:              "jamulus-headless_3.9.0_ubuntu_amd64.deb"
   deb-gui-armhf:             "jamulus_3.9.0_ubuntu_armhf.deb"
   deb-headless-armhf:        "jamulus-headless_3.9.0_ubuntu_armhf.deb"
+  deb-gui-arm64:             "jamulus_3.9.0_ubuntu_arm64.deb"
+  deb-headless-arm64:        "jamulus_3.9.0_ubuntu_arm64.deb"
```

Do we not produce two separate debs?
```suggestion
  deb-headless-arm64:        "jamulus-headless_3.9.0_ubuntu_arm64.deb"
```

---

Thanks. Fixed.